back-end/apis/image.py

```diff
@@ -179,6 +179,40 @@ def get_split_length(split):
 
     return RResponse.ok(response)
 
+@app.route('/image/classified/<split>', methods = ['GET'])
+def get_classfied_split_length(split):
+    """
+    Gets the length of all/correctly classified/incorrectly classified split lengt
+    ---
+    tags:
+      - image
+    parameters:
+      - name: "split"
+        in: "path"
+        description: "name of the split"
+        required: true
+        type: "string"
+    responses:
+      200:
+        description: The length of the split
+        schema:
+          properties:
+            code:
+              type: integer
+              example: 0
+            data:
+              type: string
+              example: 9468
+            msg:
+              type: string
+              example: Success
+    """
+    try:
+      response = getClassifiedSplitLength(split)
+    except Exception:
+      return RResponse.fail("Split not supported")
```

Comment:
In the latest dev branch, `RResponse.fail` is no longer used. Please merge the latest dev branch and use `RResponse.abort` instead.